sjb/generated/push_cluster_operator_images.xml

```diff
@@ -0,0 +1,532 @@
+scp -F ${WORKSPACE}/.config/origin-ci-tool/inventory/.ssh_config /var/lib/jenkins/.docker/co-config.json openshiftdevel:/tmp/.docker/config.json</command>
+        </hudson.tasks.Shell>
+        <hudson.tasks.Shell>
+          <command>#!/bin/bash
+SCRIPT_START_TIME=&#34;$( date +%s )&#34; &amp;&amp; export SCRIPT_START_TIME &amp;&amp; echo &#34;########## STARTING STAGE: PUSH THE CLUSTER OPERATOR IMAGES ##########&#34; &amp;&amp; trap &#39;export status=FAILURE&#39; ERR &amp;&amp; trap &#39;set +o xtrace; SCRIPT_END_TIME=&#34;$( date +%s )&#34;; ELAPSED_TIME=&#34;$(( SCRIPT_END_TIME - SCRIPT_START_TIME ))&#34;; echo &#34;########## FINISHED STAGE: ${status:-SUCCESS}: PUSH THE CLUSTER OPERATOR IMAGES [$( printf &#34;%02dh %02dm %02ds&#34; &#34;$(( ELAPSED_TIME/3600 ))&#34; &#34;$(( (ELAPSED_TIME%3600)/60 ))&#34; &#34;$(( ELAPSED_TIME%60 ))&#34; )] ##########&#34;&#39; EXIT &amp;&amp; set -o errexit -o nounset -o pipefail -o xtrace &amp;&amp; if [[ -s &#34;${WORKSPACE}/activate&#34; ]]; then source &#34;${WORKSPACE}/activate&#34;; fi
+script=&#34;$( mktemp )&#34;
+cat &lt;&lt;SCRIPT &gt;&#34;${script}&#34;
+#!/bin/bash
+set -o errexit -o nounset -o pipefail -o xtrace
+cd &#34;\${GOPATH}/src/github.com/openshift/cluster-operator&#34;
+sudo chmod a+rw /tmp/.docker/config.json
+export DOCKER_CONFIG=/tmp/.docker
+
+docker tag cluster-operator:canary registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator:latest
+docker push registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator:latest
+
+docker tag cluster-operator-ansible:canary registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator-ansible:latest
+docker push registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator-ansible:latest
+
+docker tag cluster-operator-ansible:v3.10 registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator-ansible:v3.10
+docker push registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator-ansible:latest
+
+docker tag cluster-operator-ansible:v3.9 registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator-ansible:v3.9
+docker push registry.svc.ci.openshift.org/openshift-cluster-operator/cluster-operator-ansible:latest
+
+docker tag clusterapi:latest registry.svc.ci.openshift.org/openshift-cluster-operator/kubernetes-cluster-api:latest
+docker push registry.svc.ci.openshift.org/openshift-cluster-operator/kubernetes-cluster-api:latest
+SCRIPT
+chmod +x &#34;${script}&#34;
+scp -F ${WORKSPACE}/.config/origin-ci-tool/inventory/.ssh_config &#34;${script}&#34; openshiftdevel:&#34;${script}&#34;
+ssh -F ${WORKSPACE}/.config/origin-ci-tool/inventory/.ssh_config -t openshiftdevel &#34;bash -l -c \&#34;timeout 7200 ${script}\&#34;&#34; </command>
+        </hudson.tasks.Shell>
+  </builders>
+  <publishers>
+    <org.jenkinsci.plugins.postbuildscript.PostBuildScript plugin="postbuildscript@0.17">
```

Comment:
Rebase and regenerate this xml, the postbuildscript has breaking changes that are in the updated template.

Reply:
thx will do

Reply:
done